auction-app-frontend/src/pages/Home/Home.jsx

```diff
@@ -1,7 +1,138 @@
-import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+import {
+  ProductGrid,
+  ErrorComponent,
+  LoadingComponent,
+  Button,
+} from "src/components";
+
+import {
+  useCategoriesPaginated,
+  useProductsPaginated,
+  useProduct,
+} from "src/hooks";
+
+import { go } from "src/assets/icons";
+
+import "./style.scss";
 
 const Home = () => {
-  return <div>Home Page</div>;
+  const [activeTab, setActiveTab] = useState("newArrivals");
+  const [page, setPage] = useState(0);
+  const [items, setItems] = useState([]);
+  const [hasMore, setHasMore] = useState(true);
+
+  const {
+    product,
+    loading: productLoading,
+    error: productError,
+  } = useProduct(31); // hardcoded id for now
```

Comment:
For this case, you can make an API on the backend that will return random product and use it for now, and later we can update it to highlighted product that will be fetched by the specific parameters. 
You can't use getProductById, because you will have to keep it hardcoded all the time.